src/app/functions/server/getOneRepResult.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export async function getOneRepResult(
+  method: string,
+  path: string,
+  body?: string
+) {
+  const bearerToken = process.env.ONEREP_API_KEY;
+  const options = {
+    method,
+    headers: {
+      Authorization: `Bearer ${bearerToken}`,
+      "Content-Type": "application/json",
+    },
+  };
+
+  if (body) {
```

Comment:
One subtle thing we changed here was the `method` check above from src/app/api/v1/user/welcome/route.ts#L23

```diff
-if (method !== "GET" && method !== "HEAD") {
+if (body) {
```

is `POST` the only valid method that supports `body` payloads, and do we want to add some check back in to make sure somebody isn't accidentally trying to pass a body along w/ a `GET` request?